test/functional/template_test.rb

```diff
@@ -97,4 +97,27 @@
       assert_cmd(success_result("Provisioning template updated\n"), result)
     end
   end
+
+  describe 'create' do
+    before do
+      @cmd = %(template create)
```

Comment:
@rahulbajaj0509 your `@cmd` is string, which makes the tests fail on "no implicit conversion of Array into String (TypeError)".
It should have been `@cmd = %w(template create)`